src/main/java/org/sonarlint/intellij/analysis/SonarLintTask.java

```diff
@@ -167,64 +169,14 @@ private static void checkCanceled(ProgressIndicator indicator, Project project)
 
     LOGGER.info(indicator.getText());
 
-    CancelMonitor monitor = new CancelMonitor(indicator, Thread.currentThread());
+    ProgressMonitor progressMonitor = new TaskProgressMonitor(indicator);
     List<AnalysisResults> results = new LinkedList<>();
 
-    try {
-      monitor.start();
       for (Map.Entry<Module, Collection<VirtualFile>> e : job.filesPerModule().entrySet()) {
-        results.add(analyzer.analyzeModule(e.getKey(), e.getValue(), listener));
+        results.add(analyzer.analyzeModule(e.getKey(), e.getValue(), listener, progressMonitor));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 7. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)